azure/v12.0.1/README.md

```diff
@@ -0,0 +1,15 @@
+## :zap:  Giant Swarm Release 12.0.1 for Azure :zap:
+
+**If you are upgrading from 12.0.0, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades external-dns app to v1.3.0.
+
+**Notes for future 12.x.x releases:**
+In order to proceed with upgrade, clusters must be using 11.4.0 or newer release version, otherwise the upgrade request will fail.
```

Comment:
What is this about @sslavic can we provide context?

Reply:
Azure 12+ has nginx optional, but requires that preparatory migration (change of nginx LoadBalancer Service ownership from azure-operator to nginx IC app) work is applied first, there are manual steps, as documented in 11.4.0 release notes. There's even automation in place, admission controller on Azure, ensuring that one cannot skip over multiple major or minor releases. So customer cannot upgrade from say 11.3.3 to 12.0.1, has to upgrade to 11.4.0 first. And this is just note/reminder copied from 12.0.0 release notes. Not sure if we have to preserve it until all customer clusters are upgraded to Azure 12+. It was easier to add, and remove if not needed.

@giantswarm/team-celestial what say you, should we keep this note or rely that SEs and customers read it in 12.0.0 release notes?

Reply:
Will remove it, automation prevents skipping multiple major or minor releases